### DISCUSSION TITLE
essay/2023/10/16/osamu

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#137] [Comments]
# essay/2023/10/16/osamu

개발자는 어쩌면 크리에이터가 아닐까 생각하게 된 이후로… 크리에이터 거장들이 하는 이야기와 작품들을 즐겨 보고 듣습니다.

https://jeho.page/essay/2023/10/16/osamu.html

**Comment by toyz2020** [comment]
뼈때리는 말들이군요. 완전 공감합니다.

**Comment by yuhaeun-la** [comment]
하하하하 글이 너무 재밌어요. 특강때 개발자는 크리에이터란 말을 해주신게 기억에 남아서 블로그 글을 이것저것 읽고 있습니다. 크리에이터들이(개발자 포함) 하는 고민들은 아주 유구하고, 답은 일맥상통하네요 ㅎㅎ

**Reply by BenjaminKim** [reply]
댓글이 달려서 저도 다시 읽어 봤는데 정말 좋은 조언들이네요. 영향을 많이 받은 것 같아요.
댓글 남겨주셔서 감사합니다. 자주 놀러오세요!
